### DISCUSSION TITLE
暴力的な言葉などが、コードに入っている場合BANになりますか？

### DISCUSSION THREAD

**Discussion by ghost** [community/community#83786] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

自分はつい最近マインクラフトのチャットでのNGワードを検出するプラグインを作ったのですが、YMLファイルに、NGワードとして検出するための文字列（暴力的な内容な性的な内容）をgitHUBにアップロードしたのですが、場合BANの対象になりますか？

**Comment by Akash1134** [comment]
Hi @SAKURAMOTI11151 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
